### PR TITLE
Add environment variable to disable block ingestor

### LINKED ISSUE #682: Add environment variable to disable block ingestion

### PR DESCRIPTION
**Pull request by timmclean**
Resolves #682

### CODE REVIEW

File: node/src/main.rs [hunk 1]
```diff
@@ -399,18 +399,20 @@ fn async_main() -> impl Future<Item = (), Error = ()> + Send + 'static {
...
-    // Create Ethereum block ingestor
-    let block_ingestor = graph_datasource_ethereum::BlockIngestor::new(
-        store.clone(),
-        transport.clone(),
-        50, // ancestor count, which we could make configuable
-        logger.clone(),
-        block_polling_interval,
-    )
-    .expect("failed to create Ethereum block ingestor");
+    if env::var_os("DISABLE_BLOCK_INGESTOR").unwrap_or("".into()) != "true" {
```
Comment (leodasvacas) [inline node/src/main.rs]: We can just test if the var is present at all, what do you think?

**Review by leoyvens — COMMENTED** [review]
I have a suggestion. Nice to see that the ingestor is 100% self contained.

File: node/src/main.rs (hunk 1 above)
Comment (timmclean) [reply node/src/main.rs]: Ah, I did it this way in case someone tries to do `DISABLE_BLOCK_INGESTOR=false` actually

Comment (leodasvacas) [reply node/src/main.rs]: Makes sense, though there's also the case of trying `DISABLE_BLOCK_INGESTOR=1` or `DISABLE_BLOCK_INGESTOR=yes` or something. But either way works.

**Review by leoyvens — APPROVED** [review]

File: node/src/main.rs (hunk 1 above)
Comment (timmclean) [reply node/src/main.rs]: Yeah, that's true... :smile:

**Pull request merged**